spec/negotiation.html

```diff
@@ -304,6 +301,36 @@ <h1>SupportedLocales ( _availableLocales_, _requestedLocales_, _options_ )</h1>
       </emu-alg>
     </emu-clause>
 
+    <emu-clause id="sec-normalizeoptionsobject" aoid="NormalizeOptionsObject">
+      <h1>NormalizeOptionsObject ( _options_ )</h1>
```

Comment:
Only in 262, right?

Reply:
I put it in here initially as per https://github.com/tc39/ecma402/issues/480#issuecomment-679420266